pkg/controller/cluster/cluster_controller_test.go

```diff
@@ -218,10 +220,10 @@ func processSync(c *ClusterController, key string) error {
 
 // validateClientActions validates that the client experienced the specified
 // expected actions.
-func validateClientActions(t *testing.T, clusterOperatorClient *clusteroperatorclientset.Clientset, expectedActions ...expectedClientAction) {
+func validateClientActions(t *testing.T, name string, clusterOperatorClient *clusteroperatorclientset.Clientset, expectedActions ...expectedClientAction) {
```

Comment:
I don't understand why we need to pass the test name in here. The output is already grouped by the test/subtest.

Reply:
And if we really, *really* want to pass in the name, let's at least call the parameter testName.

Reply:
In the output I was only getting the line of the failure and the message. Because we use a common validation function, it was hard to tell which test/testcase had caused it. I will update the parameter name.